### PR TITLE
modified Json

### PR DESCRIPTION
**Pull request by Vidushi-GitHub**

### CODE REVIEW

File: core-schema/gcn.notices.core.AlertReporter.json [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
             "description": "Name of Mission or Telescope reporting the event"
         },
         "instrument": {
-            "type": ["string"],
+            "type": "string",
             "description": "Name of the Instrument reporting the event"
         },
         "record_number": {
-            "type": ["int"],
+            "type": "int",
```
Comment (Vidushi-GitHub) [inline core-schema/gcn.notices.core.AlertReporter.json:17]: "int" will be replaced

File: core-schema/gcn.notices.core.HardnessRatio.json [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
     },
-    "energy_range_hard": {
-      "$ref": "gcn.notices.core.HardnessRatio",
-      "description": "Energy range [keV] used in hard band of hardness ratio"
-    },
-    "energy_range_soft": {
-      "$ref": "gcn.notices.core.HardnessRatio",
-      "description": "Energy range [keV] used in soft band of hardness ratio"
+    "energy_range": {
+      "$ref": "gcn.notices.core.EventInfo.json",
```
Comment (Vidushi-GitHub) [inline core-schema/gcn.notices.core.HardnessRatio.json:13]: By mistake have called "EventInfo" instead of EnergyRange

**Review by lpsinger — APPROVED** [review]

File: core-schema/gcn.notices.core.HardnessRatio.json (hunk 2 above)
Comment (lpsinger) [reply core-schema/gcn.notices.core.HardnessRatio.json:13]: OK. I've approved this. Merge it whenever you are ready.

**Comment by lpsinger** [comment]
@Vidushi-GitHub, are you merging this?

**Comment by Vidushi-GitHub** [comment]
> I want to merge, it seems like I don't have permission to merge here. 
Other way is terminal, git checkout master -> git pull origin master -> git merge core-schema -> git push origin master. That may create conflicts.

**Pull request merged**